code/modules/admin/watchlist.dm

```diff
@@ -142,6 +142,6 @@
 		output += "<b>[ckey]</b> | Added by <b>[adminckey]</b> on <b>[timestamp]</b> <a href='byond://?_src_=holder;watchremovebrowse=[ckey]'>\[Remove\]</a> <a href='byond://?_src_=holder;watcheditbrowse=[ckey]'>\[Edit Reason\]</a>"
 		if(last_editor)
 			output += " <font size='2'>Last edit by [last_editor] <a href='byond://?_src_=holder;watcheditlog=[ckey]'>(Click here to see edit log)</a></font>"
-		output += "<br>[reason]<hr style='background:#000000; border:0; height:1px'>"
+		output += "<html><meta charset='utf-8'><br>[reason]<hr style='background:#000000; border:0; height:1px'>"
```

Comment:
Hmm, is there any way that we might be able to put this at the top of the document instead?

Reply:
If I had known...